library/src/main/java/com/mapzen/android/MapzenMap.java

```diff
@@ -676,4 +708,26 @@ public void drawTransitRouteLine(@NotNull List<LngLat> points, @Nullable List<Ln
   public void clearTransitRouteLine() {
     overlayManager.clearTransitRouteLine();
   }
+
+  /**
+   * By default all {@link MapData} objects are removed from the map when it is destroyed. To
+   * persist this data such as in the case of orientation changes, use this method.
+   * @param persistOnRecreation persist {@link MapData} across orientation changes
+   */
+  public void setPersistMapData(boolean persistOnRecreation) {
```

Comment:
Whoops, just seeing the second comment after refreshing and want to clarify- we want to persist the map state on default but not the map data, right?

Separately, we should update the sample app to persist data so that it doesnt feel broken